exercises/tmclass_exercises/text_manipulation.py

```diff
@@ -67,9 +69,19 @@ def tokenize_generic(text):
     # - `unicodedata.category(c)` returns the categoriy of character `c`
     # - The list of categories is available at:
     # http://www.unicode.org/reports/tr44/tr44-6.html#General_Category_Values
-
-    # TODO: write me!
-    return []
+    L = ["L", "M", "N"]
+    result = []
+    mot = ""
```

Comment:
Il est préférable que les noms des variables soient en anglais.

Reply:
ok, I will resolve it after. Thanks